src/main/resources/templates/signIn.html

```diff
@@ -73,15 +74,20 @@
     <!-- content of main -->
 
     <div class="container">
-      <div class="content-sign-in content">
+      <div class="content-sign-in">
+
+        <div class="user__message">
+          <div th:if="${verified != null}" th:text="${verified}"></div>
+          <div th:if="${incorrectCredentials != null}" th:text="${incorrectCredentials}"></div>
```

Comment:
If we will use word "message" as a key, you can leave only one check for condition